src/app/Peerly/screens/GiveAppreciationScreen/index.tsx

```diff
@@ -39,7 +39,8 @@ const paginationData = {
 const schema = yup.object().shape({
   receiver: yup.string().required(messages.SELECT_COWORKER_NAME),
   core_value_id: yup.string().required(messages.SELECT_CORE_VALUE),
-  description: yup.string().required(messages.ENTER_DESCIPTION),
+  description: yup.string().required(messages.ENTER_DESCIPTION)
+  .min(150, 'Description must be at least 150 characters long.'),
```

Comment:
Can we write message of line no 43 in message constant file.

Reply:
Kindly attach screen shot which display error message as well.

Reply:
![1000139324](https://github.com/user-attachments/assets/5f3d0bfa-625c-4a1e-a7c4-86d835ee5a14)